tuna/miopen/utils/metadata.py

```diff
@@ -220,6 +220,7 @@
     'FP16': 'FP16',
     'convfp16': 'FP16',
     'BF16': 'BF16',
+    'BFP16': 'BF16',
```

Comment:
Is it anticipated that Tuna encounters BFP16 as input? I thought it was intended to eliminate this.